src/pudl/transform/eia923.py

```diff
@@ -700,9 +700,9 @@ def generation_fuel_eia923(raw_generation_fuel_eia923: pd.DataFrame):
     # Aggregate any remaining duplicates.
     gen_fuel = _aggregate_generation_fuel_duplicates(gen_fuel)
 
-    return Output(output_name="clean_generation_fuel_eia923", value=gen_fuel), Output(
-        output_name="clean_generation_fuel_nuclear_eia923",
-        value=generation_fuel_nuclear_eia923,
+    return (
+        Output(output_name="clean_generation_fuel_eia923", value=gen_fuel),
+        Output(output_name="clean_generation_fuel_nuclear_eia923", value=gen_fuel_nuke),
```

Comment:
Nothing substantial going on here. Just used a shorter variable name to get stuff on one line.